### DISCUSSION TITLE
Poll Context object for API changes overwriting the batching changes

### DISCUSSION THREAD

**Discussion by palaniappa** [confluentinc/parallel-consumer#262] [General] [answered]
Hi @astubbs 

I see that the commit 7c5ac01 has overwritten the pollBatch or batching changes. Is this intentional?

Thanks
Palani

**Comment by astubbs** [comment, marked as answer]
Hi! Yes, the next version has a different and improved batching system. Now all api access points support batching. There is no need for a dedicated batching end point. 

See https://github.com/confluentinc/parallel-consumer#batching

Please if you have other concerns, let me know. Or if the new ReadMe section on batching isn't clear, let me know. 

I didn't include anything regrading this in the upgrade section as the pollbatch api was never in a release. 

https://github.com/confluentinc/parallel-consumer#from-0-4-to-0-5

**Reply by astubbs** [reply]
Also let me know if you have input on the new API!
